packages/formik-effect/src/tests/index.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { mount } from 'enzyme';
+import { Field, Form, Formik } from 'formik';
+import React from 'react';
+import FormikEffect from '..';
+
+describe('formik-effect', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('works as expected', () => {
+    const mock = jest.fn();
+
+    const SignupForm = () => (
+      <div>
+        <h1>My Cool Form with a SideEffect</h1>
+        {/* tslint:disable jsx-no-lambda */}
+        {/* tslint:disable no-console */}
+        <Formik
+          onSubmit={values => console.log(values)}
```

Comment:
stow away console

Reply:
Maybe we could use a mock

Reply:
This one was intentional, but let me remove it.